openapi/components/schemas/dateTimeString.yaml

```diff
@@ -0,0 +1,4 @@
+title: Datetime
+description: An RFC3339 string
```

Comment:
Can we provide an example of the format of this string.  As we have discussed before RFC3339 string does not help anyone really know what this will actually contain.  Does it have timezone, is it zulu time, does it have microseconds.. etc. .etc..